eslint not found

```sh
docker-compose -f docker-compose.yml run app sh -c "black -l 90 .&&yarn workspace @experimenter/core lint-fix&&yarn workspace @experimenter/rapid lint:eslint --fix"
Starting experimenter_redis_1 ... done
Starting experimenter_db_1    ... done
All done! ✨ 🍰 ✨
230 files left unchanged.
yarn workspace v1.22.4
yarn run v1.22.4
$ eslint --fix js/components/**/*.js js/tests/**/*.js js/index.js
/bin/sh: 1: eslint: not found
error Command failed with exit code 127.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed.
Exit code: 127
Command: /usr/bin/node
Arguments: /usr/share/yarn/lib/cli.js lint-fix
Directory: /app/experimenter/static/core
Output:

info Visit https://yarnpkg.com/en/docs/cli/workspace for documentation about this command.
make: *** [code_format] Error 127
```

--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -12,6 +12,8 @@ services:
     volumes:
       - ./app:/app
       - /app/node_modules/
+      - /app/experimenter/static/core/node_modules/
+      - /app/experimenter/static/rapid/node_modules/
       - assets_volume:/app/experimenter/static/assets/
       - static_volume:/app/experimenter/served/
       - /app/.cache

